lib/tasks/osc.rake

```diff
@@ -159,12 +159,17 @@ namespace :osc do
       Dir.chdir osc_checkout_dir do
         puts "building package..." if verbose
 
-        # pipe yes to osc build to automatic rebuild broken build root if it happen
-        command = "yes | osc -A '#{obs_api}' build"
+        # have separated roots per target system, so sharing is more efficient
+        broot = "/var/tmp/build-root-#{build_dist}"
+        command = "osc -A '#{obs_api}' build"
         command << " --no-verify" #ignore untrusted BS projects
         command << " --release=1" #have always same release number
         # have separated roots per target system, so sharing is more effficient
-        command << " --root=/var/tmp/build-root-#{build_dist}"
+        command << " --root=#{broot}"
+        # avoid an interactive question, bsc#1053839
+        if File.exist?("#{broot}/not-ready")
```

Comment:
ugh, what? who create file? where it is documented? any documentation for that not-ready file would be good. Do not forget that this packaging rake tasks is used for multiple projects, not only libstorage or yast.

Reply:
I think I see your point.

This PR replaces a `yes |` workaround by something which is still a workaround. `not-ready` is an [implementation detail of `init_buildsystem`](https://github.com/openSUSE/obs-build/blob/bfdadae5c2b78ff6aafc4d0bf5f7682a0ffbc7e6/init_buildsystem#L134), which is part of build.rpm, on which `osc` depends.

I considered `init_buildsystem`  a more stable thing than `osc`. Do you think we should press to have https://bugzilla.suse.com/show_bug.cgi?id=1053839 fixed instead?

Reply:
upstream fixes is always nice to have :), but if you document this, I am, also fine with this File exists stuff